btsieve/src/bitcoin/mod.rs

```diff
@@ -31,7 +31,7 @@ where
 
     fn matching_transactions(
         &self,
-        query: TransactionQuery,
+        query: Pattern,
```

Comment:
```suggestion
        pattern: Pattern,
```

Reply:
Cool, done.  This raises an issue that this PR only touches btsieve - all through cnd we still use the term 'query'.  I intend on opening an issue for that.